docs/manual/interning-checker.tex

```diff
@@ -118,9 +126,11 @@
   for such a class, \<==> and \<equals> are equivalent, and so the
   Interning Checker does not issue errors or warnings for either one.
 
+\item[\refqualclass{checker/interning/qual}{InternMethod}]
+  is a method annotation that indicates that this method returns an interned object and maybe be invoked
```

Comment:
Does "method annotation" have a well-accepted definition? I would have said "declaration annotation" and then mention elsewhere that it should only be used on methods.